features/unattached_commands.feature

```diff
@@ -152,8 +152,8 @@ Feature: Command behaviour when unattached
         Then stdout matches regexp:
             """
             USN-4539-1: AWL vulnerability
-            https://ubuntu.com/security/notices/USN-4539-1
-            Related CVEs: CVE-2020-11728.
+            Found CVEs: CVE-2020-11728.
```

Comment:
are we sure about the periods? This seems odd since it's not a sentence?

Reply:
+1 dropping the trailing stop from  this output.